_layouts/workshop.html

```diff
@@ -9,10 +9,26 @@
   <head>
     <title>Software Carpentry: {{ page.venue }}</title>
     {% include header.html %}
-    <meta name="venue" content="{{page.venue}}" />
-    <meta name="date" content="{{page.humandate}}" />
+    <meta name="slug" content="{{site.github.project_title}}" />
     <meta name="startdate" content="{{page.startdate}}" />
     <meta name="enddate" content="{{page.enddate}}" />
+    <meta name="country" content="{{page.country}}" />
+    <meta name="venue" content="{{page.venue}}" />
+    <meta name="address" content="{{page.address}}" />
+    <meta name="latlng" content="{{page.latlng}}" />
+    <meta name="language" content="{{page.language}}" />
+    <meta name="eventbrite" content="{{page.eventbrite}}" />
+    {% if page.instructor == "FIXME" %}
```

Comment:
This feels fragile to me - I'm 99% sure that some people will (for example) leave `instructor` blank rather than putting in `FIXME`.